README.md

```diff
@@ -422,6 +468,24 @@ type Keys = OptionalKeys<Props>;
 
 [⇧ back to top](#table-of-contents)
 
+### `PossibleKeys<T>`
+
+Similar to [`$Keys`](#keyst) or `keyof`, but get keys also from union types
+
+**Usage:**
+
+```ts
+type Props = { name: string; employeeId: string } | { name: string; guestId: string };
+// Expect: "name" | "employeeId" | "guestId"
+type PropsKeys1 = PossibleKeys<Props>;
+// Expect: "name"
+type PropsKeys2 = $Keys<Props>;
+// Expect: "name"
+type PropsKeys3 = keyof Props;
+```
+
+[⇧ back to top](#table-of-contents)
+
```

Comment:
we already have another proposal for this type called UnionKeys.
I layed out specs there which are not respected here so I propose either:
- update this PR according to specs
- remove this type from this PR